tests/utils/hints.py

```diff
@@ -53,3 +53,33 @@ def patch_hint(program, hint, new_hint):
         raise ValueError(f"Hint\n\n{hint}\n\nnot found in program hints.")
     with patch.object(program, "hints", new=patched_hints):
         yield
+
+
+@contextmanager
+def insert_hint(program, location: str, hint):
```

Comment:
thats cool!

Reply:
All credits to @ClementWalter !
But indeed very cool